### PR TITLE
Refine _get_config_dict() and clean error messages

### PR DESCRIPTION
**Pull request by MHA3**

### CODE REVIEW

File: src/clusterfuzz/_internal/build_management/overrides.py [hunk 1]
```diff
@@ -23,9 +23,10 @@
 PLATFORM_ID_TO_BUILD_PATH_KEY = 'build_paths'
 PLATFORM_ID_TO_REV_PATH_KEY = 'revisions_paths'
 
-OVERRIDE_PATH_NOT_FOUND_ERROR = 'Could not find override path from config.'
-OVERRIDE_CONFIG_NOT_JSON_ERROR = 'Could not read config as json.'
-OVERRIDE_CONFIG_NOT_READ_ERROR = 'Could not import config file.'
+OVERRIDE_PATH_NOT_FOUND_ERROR = 'Could not find override path from config, '\
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/build_management/overrides.py:26]: Please use ( ) for line continuation instead per https://google.github.io/styleguide/pyguide.html#32-line-length.

Comment (MHA3) [reply src/clusterfuzz/_internal/build_management/overrides.py]: done

**Review by oliverchang — APPROVED** [review]

**Pull request merged**